sources/packages/backend/libs/integrations/src/services/disbursement-schedule/disbursement-schedule.service.ts

```diff
@@ -68,71 +70,23 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
       .getMany();
   }
   /**
-   * Fetch the COEs which are pending for the institution.
-   * @returns eligible COEs.
+   * Fetch the COEs which are eligible to confirm enrolment by the institutions.
+   * @returns eligible COE .
    */
-  async getPendingCOEs(): Promise<DisbursementSchedule[]> {
-    return this.repo.find({
-      select: {
-        id: true,
-        disbursementDate: true,
-        disbursementValues: { id: true, valueCode: true, valueAmount: true },
-        studentAssessment: {
-          id: true,
-          application: {
-            id: true,
-            applicationNumber: true,
-            studentNumber: true,
-            currentAssessment: {
-              id: true,
-              offering: {
-                id: true,
-                studyStartDate: true,
-                studyEndDate: true,
-                institutionLocation: {
-                  institutionCode: true,
-                },
-              },
-            },
-            student: {
-              id: true,
-              birthDate: true,
-              sinValidation: { id: true, sin: true },
-              user: { id: true, lastName: true, firstName: true },
-            },
-          },
-        },
-      },
-      relations: {
-        disbursementValues: true,
-        studentAssessment: {
-          application: {
-            currentAssessment: { offering: { institutionLocation: true } },
-            student: {
-              sinValidation: true,
-              user: true,
-            },
-          },
-        },
-      },
-      where: {
-        coeStatus: COEStatus.required,
-        studentAssessment: {
-          application: {
-            applicationStatus: In([
-              ApplicationStatus.Enrolment,
-              ApplicationStatus.Completed,
-            ]),
-            currentAssessment: {
-              offering: {
-                institutionLocation: { hasIntegration: true },
-                offeringIntensity: OfferingIntensity.fullTime,
-              },
-            },
-          },
-        },
-        hasEstimatedAwards: true,
-      },
-    });
+  async getInstitutionEligiblePendingEnrolments(): Promise<
+    DisbursementSchedule[]
+  > {
+    const eligibleCOEQuery =
+      this.confirmationOfEnrollmentService.getDisbursementForCOEQuery(
+        this.repo,
+        true,
+      );
+    return eligibleCOEQuery
+      .andWhere("offering.offeringIntensity = :fullTime", {
+        fullTime: OfferingIntensity.fullTime,
+      })
+      .andWhere("location.hasIntegration = TRUE")
```

Comment:
Can we have the `TRUE` as a lower case following the `getDisbursementForCOEQuery` query?